sqlstore/src/main/resources/db/migration/V8801__library_size.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE Library ADD COLUMN `size` bigint(20);
+
+UPDATE Library SET size = (SELECT insertSize FROM LibraryQC WHERE LibraryQC.library_libraryId = Library.libraryId ORDER BY qcDate LIMIT 1);
```

Comment:
Did not realize that a previous migration preserves old insert sizes separately, so the data will never be removed automatically during flyway migration, so this assumption is okay